dynamixel_controllers/src/dynamixel_controllers/joint_trajectory_action_controller.py

```diff
@@ -100,6 +100,9 @@ def initialize(self):
         for joint in self.joint_names:
             self.goal_constraints.append(rospy.get_param(ns + '/' + joint + '/goal', -1.0))
             self.trajectory_constraints.append(rospy.get_param(ns + '/' + joint + '/trajectory', -1.0))
+
+        self.cancel_prev_goals_on_preemption = \
+            rospy.get_param(self.controller_namespace + '/cancel_prev_goals_on_premption', True)
```

Comment:
Setting this param to False mimics position_controller behavior.
Setting it to True maintains backwards compatibility, and mimics velocity and effort control behavior.